SimPEG/Maps.py

```diff
@@ -465,10 +465,6 @@ class SelfConsistentEffectiveMedium(IdentityMap, properties.HasProperties):
 
             \chi = \sqrt{\\frac{1}{\\alpha^2} - 1}
```

Comment:
It will be nice clarify what are the input and output parameters of the mapping
So, input is simga0, sigma1, phi, and output is effective sigma

Also, it will be nice to comment what is the inversion parameters in mind for the current set up.

Reply:
Thanks seogi, I have specified it in the docs above